fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -501,7 +501,8 @@ public List<Long> getMvColumnIndexIds(String columnName) {
         for (MaterializedIndexMeta meta : getVisibleIndexIdToMeta().values()) {
             Column target = meta.getColumnByDefineName(columnName);
             if (target != null) {
-                ids.add(meta.getIndexId());
+                // Return -1 for base index. This is for compatible with older version of column stats.
+                ids.add(meta.getIndexId() == baseIndexId ? -1 : meta.getIndexId());
```

Comment:
It seems that this interface is specifically designed for statistical information. Please add some comments to clarify its purpose and usage.

Reply:
added